plugins/woocommerce/templates/loop/result-count.php

```diff
@@ -14,14 +14,14 @@
  *
  * @see         https://woocommerce.com/document/template-structure/
  * @package     WooCommerce\Templates
- * @version     9.4.0
+ * @version     9.9.0
  */
 
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
 ?>
-<p class="woocommerce-result-count" <?php echo ( empty( $orderedby ) || 1 === intval( $total ) ) ? '' : 'role="alert" aria-relevant="all" data-is-sorted-by="true"'; ?>>
+<p class="woocommerce-result-count" role="alert" aria-relevant="all">
```

Comment:
I am not opposed to removing this condition, but would like to know the exact reason you decided to remove it?
As I am not sure what the exact context was when it was added.

Reply:
@louwie17 

Both the `role="alert"` and `aria-relevant="all"` attributes cannot be under that condition so the result count is announced even on the first page load. However, the `data-is-sorted-by="true"` attribute needs to be under the condition. Thanks for catching that! 

I have updated the code: [Conditionally add data-is-sorted-by to result count element](https://github.com/woocommerce/woocommerce/pull/55338/commits/24e3ff342af84cbea6780e3d706bd1fcbabba66e)